### ISSUE TITLE
Terminal issues when connected to VPN

### ISSUE THREAD

**Issue by NI6HT0WL** [theGuildHall/pwnbox#15]
When I connect to the VPN for HTB, I open a new terminal. It'll show as a blank terminal until I press ctrl+c, then I'm able to start using the terminal. However when I do this, it doesn't continue using the prompt colours set in .bashrc either.
Terminal colours:
![image](https://user-images.githubusercontent.com/73410683/97112389-ecb70b80-16db-11eb-91f3-d41910666ed4.png)

Terminal after ctrl+c once connected to VPN:
![image](https://user-images.githubusercontent.com/73410683/97112415-0a847080-16dc-11eb-8a9a-1334f4f141b4.png)

**Comment by marius56** [comment]
tl;dr;
Copy your htb openvpn configuration file to `/etc/openvpn/htb_username.conf` (**Note**: Extension `.conf` is required!) and start the vpn with `systemctl start openvpn`. To stop it use `systemctl stop openvpn`.

Hey,
I had the same issue. My mistake was to start the vpn connection in a terminal. The cause is the file /opt/vpnserver.sh which tries to get the server you are connected to (e.g. `eu-free-3`).

It does so with the following line:

``vpn=$(cat `systemctl status openvpn@* | grep "/usr/sbin/openvpn" | tr -s " " | cut -d " " -f 12` | grep "remote " | cut -d " " -f 2)``

The script needs the active openvpn configuration file and is using `systemctl status openvpn@*` to show the details of the openvpn service. But if openvpn was started in a terminal (e.g. with `openvpn htb_user.ovpn`), there are no details and the script hangs..

To solve it, just copy your htb openvpn configuration file to `/etc/openvpn/htb_user.conf` (**Note**: The extention has to be changed to `.conf`!). The openvpn service will use the .conf file by default when it starts.
You can then activate the vpn connection with `systemctl start openvpn` and stop it with `systemctl stop openvpn`

**Comment by theGuildHall** [comment]
Great reply marius56! I haven't had much time to dig into the issue opened.

Correct, I have updated the script to pull from `/etc/openvpn/whatever.conf` since it's cleaner and easier to accommodate for other VPN services (such as HTB Battlegrounds, tryhackme, or Offsec's proving grounds).

In fact, in my own personal VM, I have setup aliases to quickly start/stop my VPNs. Here is what I have setup:

```bash
DANTE='sudo systemctl start openvpn@dante.service'
HTB='sudo systemctl start openvpn@theguildhall.service'
PG='sudo systemctl start openvpn@pg.service'
THM='sudo systemctl start openvpn@thm.service'
stop='sudo systemctl stop openvpn@*'
```

For every new service I signup for, I create an alias for it so I can just type `DANTE` or `THM` from my terminal to connect to the VPN. And if I want to disconnect, i just type `stop`. That way, we don't need to use `openvpn file.ovpn`.

I might create a new repo with what I use versus cluttering up this repo.
